{
  "hash": "ce04c7eb706c95dd9f7378cdc26db149",
  "result": {
    "markdown": "---\ntitle: \"Package Errors and Warnings\"\nauthor: \"Sabrena Rutledge\"\ndate: \"2023-03-30\"\ncategories: \"Errors and warnings in packages\"\noutput: github_document\n---\n\n\n<!-- README.md is generated from README.Rmd. Please edit that file -->\n<!-- badges: start -->\n[![Frontmatter check](../../actions/workflows/check-yaml.yaml/badge.svg)](../../actions/workflows/check-yaml.yaml)\n<!-- badges: end -->\n\n## Prompt:\n\nFix one of the problems in one of our community packages at `https://github.com/Stat585-at-ISU/lab-3-all-all-for-one-and-one-for-all`, and write about it. \n\n## Instructions:\n\nWhat we want to know is the **exact** warning or error message of the item you fixed, and a description of what you did, also as specific as possible. \n\nThe exact warning message: Warning message: Expected 2 pieces. Missing pieces filled with `NA` in 50 rows [1, 2, 3, 4, 5, 32, 33, 34, 35, 36, 63, 64, 65, 66, 67, 94, 95, 96, 97, 98, ...].\nI referenced the separate function's documentation. Since the warning from a result from not finding 2 pieces for the separate function, I found the parameter that controls what the function does if it does not find the expected number of pieces. I then edited the line using the separate function such that the function now has a fill parameter that does not give a warning on default.\n\nDescribe your experience  below. Push this blog post to your blog-9 repo. Make sure the front matter check passes.\n\nFixing these errors was straight-forward since I knew where the errors were coming from. I have used the separate function frequently so I recognized the warning message.\n\n**Which error or warning did you fix? ... and how?**\n\nPlease see the above answer.\n\n",
    "supporting": [],
    "filters": [
      "rmarkdown/pagebreak.lua"
    ],
    "includes": {},
    "engineDependencies": {},
    "preserve": {},
    "postProcess": true
  }
}